Framework/PythonInterface/plugins/algorithms/WorkflowAlgorithms/ReflectometryISISLoadAndProcess.py

```diff
@@ -239,6 +247,28 @@ def _getInputWorkspaces(self, runs, isTrans):
             workspaces.append(ws)
         return workspaces
 
+    def _loadPolarizationCorrectionWorkspace(self):
+        efficiencies_ws = self.getPropertyValue('PolarizationEfficiencies')
+        if not efficiencies_ws:
+            return None
+
+        if AnalysisDataService.doesExist(efficiencies_ws):
+            self.log().information(f'Loading polarization efficiency information from workspace \"{efficiencies_ws}\"')
+            return AnalysisDataService.retrieve(efficiencies_ws)
+
+        args = {'InputRunList': [efficiencies_ws], 'EventMode': False}
+        alg = self.createChildAlgorithm('ReflectometryISISPreprocess', **args)
```

Comment:
It hadn't occurred to me at the time, but would it be better to just call the `LoadNexus` algorithm directly here?